flow/layers/backdrop_filter_layer.cc

```diff
@@ -43,6 +43,9 @@ void BackdropFilterLayer::Preroll(PrerollContext* context,
                                   const SkMatrix& matrix) {
   Layer::AutoPrerollSaveLayerState save =
       Layer::AutoPrerollSaveLayerState::Create(context, true, bool(filter_));
+  if (context->view_embedder != nullptr) {
```

Comment:
The filter is being pushed but never cleared. The platform view gets an ever-increasing number of filters until app is unresponsive.

Reply:
Yes, we are fixing it here: https://github.com/flutter/engine/pull/34596

Reply:
I ran with a print statement of the number of mutators in vector_ in MutatorsStack::PushBackdropFilter, it was continuously increasing